sdk/packages/selector-base/src/selector.ts

```diff
@@ -169,7 +169,9 @@ export class NCBaseSelector<A extends Adapter> {
       this._anchor.appendChild(this._modal)
```

Comment:
you can remove this file, its contents are no longer used anywhere since we use adapter structure everywhere